www/sass/patternfly.scss

```diff
@@ -26,11 +26,13 @@ limitations under the License.
 @import "node_modules/bootstrap/scss/mixins";
 
 // Core: Import Icon Fonts
+$fa-font-path:  "https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/fonts/";
 @import "node_modules/font-awesome/scss/font-awesome.scss";
+$pf-font-path:  "https://cdnjs.cloudflare.com/ajax/libs/patternfly/3.23.1/fonts/";
```

Comment:
This is unnecessary. Since we are lading the front from google then it'll be just duplicating the load.
I'll try tomorrow morning to remove it and remove something else to avoid console errors.